content/logs/s3.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Archives on AWS S3
+kind: Documentation
+description: "Forward all your Datadog logs to S3 for long term storage."
+---
+
+<div class="alert alert-warning">
+This feature is currently in Limited Beta. Ask your Sales representative or Customer Success Manager to have it enabled.
+</div>
+
+## S3 Archives
+
+Configure your Datadog account to forward all the logs ingested to your own S3 bucket in order to keep in long-term storage all the logs that you used Datadog to aggregate together. This guide shows you how to set this up. 
```

Comment:
This sentence is a bit verbose. I would say: "Configure your Datadog account to forward all the logs ingested to your own S3 bucket. This allows you to keep all the logs that you used Datadog to aggregate in long-term storage. This guide shows you how to set this up."